services/api/src/typeDefs.js

```diff
@@ -132,6 +132,13 @@ const typeDefs = gql`
     deleted: String
   }
 
+  type ProblemIdentifier {
```

Comment:
I'm a little confused by this - I think it might be the name - but is this supposed to uniquely identify a problem _across multiple projects_? That is, provide summary information?

If so, I think this needs to be reworked just a little - let's chat when you have a sec.

Reply:
Yep I think we need to chat about this - essentially this was created because returning all problems (e.g. [Problem]) isn't enough to see which projects/environments have been affected. So rather than adjust the Problem type to include project related fields - this ProblemIdentifier type was created. Since then, it might make sense to add a project field in to the Problem type. But lets chat!